src/components/molecules/BoothCard/index.tsx

```diff
@@ -1,4 +1,44 @@
 import { Typography, Box } from '@mui/material'
+import { ReactNode } from 'react'
+import { Colors } from 'src/styles/color'
+
+// Regular expression for parsing URLs (excluding closing parentheses at the end)
+const URL_REGEX: RegExp = /(https?:\/\/[^)\s)]+)/g
+
+/**
+ * Convert URL strings contained in text to links
+ * @param text text
+ */
+const replaceUrlWithLink: (text: string) => ReactNode = text => {
+  const parts = text.split(URL_REGEX)
+
+  return parts.map((part, i) => {
+    if (part.match(URL_REGEX)) {
+      return (
+        <a key={i} href={part} target="_blank" rel="noopener noreferrer" style={{ color: Colors.text.link }}>
```

Comment:
色の変化のみだとリンクであることがわからない方もいるので underline を付与した方が良さそうです。

```suggestion
        <a key={i} href={part} target="_blank" rel="noopener noreferrer" style={{ color: Colors.text.link, textDecoration: 'underline' }}>
```